clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2042,12 +2042,11 @@ private StoreSetting getSettingsFromController(ControllerClient controllerClient
         storeSetting.storeRewindTimeInSeconds = hybridStoreConfig.getRewindTimeInSeconds();
       }
     }
-
     if (setting.enableWriteCompute && !storeSetting.isWriteComputeEnabled) {
       throw new VeniceException("Store does not have write compute enabled.");
     }
 
-    if (setting.enableWriteCompute && (!storeSetting.isIncrementalPushEnabled || !setting.isIncrementalPush)) {
+    if (setting.enableWriteCompute && (!storeSetting.isHybridStore || !setting.isIncrementalPush)) {
```

Comment:
Just to confirm here the "setting.isIncrementalPush" is it coming from VPJ config?

Reply:
That’s right